pom.xml

```diff
@@ -348,6 +348,16 @@
                              metron-interface/metron-config README and NOTICE files -->
                         <exclude>dist/*.svg</exclude>
                         <exclude>dist/styles.*.bundle.css</exclude>
+                        <!-- Generated assets in the dist folder have licenses present in their source files -->
```

Comment:
We should not exclude these from RAT, but should have the licences attached to these 'binaries' using the prepend_licence_header step.